src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -648,6 +718,15 @@ bool emberAfWindowCoveringClusterUpOrOpenCallback(app::CommandHandler * commandO
     EndpointId endpoint = commandPath.mEndpointId;
 
     emberAfWindowCoveringClusterPrint("UpOrOpen command received");
+
+    EmberAfStatus status = GetMotionLockStatus(endpoint);
+    if (EMBER_ZCL_STATUS_SUCCESS != status)
+    {
+        emberAfWindowCoveringClusterPrint("Err device locked");
+        emberAfSendImmediateDefaultResponse(status);
```

Comment:
Followup: Please don't use `emberAfSendImmediateDefaultResponse`, which we are trying to get rid of.  Send responses via the command handler.

Reply:
Didn't but good to know thanks